lib/senkyoshi/models/file_resource.rb

```diff
@@ -19,5 +19,19 @@ def self.from(xml, pre_data, _resource_xids = nil)
     def iterate_xml(_xml, _pre_data)
       self
     end
+
+    def create_module(course)
+      course.canvas_modules ||= []
+      cc_module = Course.master_module(course)
```

Comment:
So course is a canvas_cc model and not ours, so I guess it makes things a little more complicated to do it that way since I don't have access to our course model to add an instance method there. I have been trying to think of a way. If you have insight let me know.